stable/mongodb-replicaset/values.yaml

```diff
@@ -122,3 +122,7 @@ livenessProbe:
   failureThreshold: 3
   periodSeconds: 10
   successThreshold: 1
+
+# postInstall:
+#   - name: job1
+#     image: mongoinit
```

Comment:
I'm working on making a service where our developers, qa, people integrating, etc. can spin up an environment. Our production environment over the years has developed a complex set of users, roles, collections, and indexes. To enforce that, we have a series of python scripts and configuration files. 

I wrote this pull request to support the flexibility I needed to run our scripts against these new databases. The example bash script I supplied was purposely simple for illustration purposes. 

I think your complexity point is valid and it would be great if someone added that functionality. But it won't suit my purposes at all.